### ISSUE TITLE
[tsrelay] error creating serve: error setting serve config: sending serve config: Access denied: serve config denied

### ISSUE THREAD

**Issue by kevmo314** [tailscale-dev/vscode-tailscale#4]
Thanks for the extension! I get this error

```
2023-06-04 20:35:30.717 [info] [tsrelay] error creating serve: error setting serve config: sending serve config: Access denied: serve config denied
2023-06-04 20:35:30.718 [info] error adding serve: Error: /serve failed
```

When running `tailscale funnel status`

```
$ tailscale funnel status

# Funnel on:
#     - https://chicago-wsl.tail21459.ts.net

Warning: funnel=on for chicago-wsl.tail21459.ts.net:443, but no serve config
         run: `tailscale serve --help` to see how to configure handlers
```

Did I miss a configuration step?

**Comment by tylersmalley** [comment]
@kevmo314, thanks for the report!

Are you running on Linux by chance?

**Comment by kevmo314** [comment]
> @kevmo314, thanks for the report!
> 
> Are you running on Linux by chance?

I'm running on Windows but connected to a WSL instance.

**Comment by tylersmalley** [comment]
@ kevmo314 we just published a release to the marketplace (0.4.3) that should resolve this issue. On Linux, we need to run the serve command with elevated permissions. So, the expected behavior is you should get a sudo prompt the first time you attempt to modify serve. Let me know if you run into issues, and thanks again for reporting!

**Comment by tylersmalley** [comment]
This is resolved. Please open a new issue if you experience this issue with the current extension.
